.ci/docker/ci_commit_pins/pytorch.txt

```diff
@@ -1 +1 @@
-a52b4e22571507abc35c2d47de138497190d2e0a
+8194fae625835394ab088985e08aabe65926e524
```

Comment:
@huydhn Put `release/2.3` directly here doesn't work. It seem like it doesn't include all newly cherry-picked PRs. Do you have any idea why?